app/commands/payments/paypal/interval.rb

```diff
@@ -0,0 +1,14 @@
+module Payments::Paypal::Interval
+  class UnsupportedPaypalIntervalError < RuntimeError; end
+
+  def self.from_payment_cycle(payment_cycle)
+    case payment_cycle
+    when 'Monthly'
+      :month
+    when 'Yearly'
+      :year
+    else
+      raise UnsupportedPaypalIntervalError
+    end
+  end
+end
```

Comment:
I'd rather this either: 
1. Followed our command naming. So module `Payments::Paypal::Interval::CycleToString.(...)`
2. Was a model in app/models/payments/paypal/interval. Probably one that was initialize_with :payment_cycle then `from_payment_cycle` was `to_s`. Or something.

This doesn't block merging to be clear, but I think it would be good to tidy/refactor to use an existing pattern.